### PR TITLE
Add support for route users

### PR DESCRIPTION
**Pull request by fenrisl**
Issue #80 

- Added new method "users" 
- Added objects models for users
- Added example script using users
- Unit test for the feature

### CODE REVIEW

File: cbw_api_toolbox/cbw_api.py [hunk 1]
```diff
@@ -13,11 +13,13 @@
 from cbw_api_toolbox.__routes__ import ROUTE_GROUPS
 from cbw_api_toolbox.__routes__ import ROUTE_PING
 from cbw_api_toolbox.__routes__ import ROUTE_REMOTE_ACCESSES
+from cbw_api_toolbox.__routes__ import ROUTE_USERS
```
Comment (fwininger) [inline cbw_api_toolbox/cbw_api.py]: Ordre alphabétique

File: cbw_api_toolbox/cbw_api.py [hunk 2]
```diff
@@ -13,11 +13,13 @@
...
 from cbw_api_toolbox.__routes__ import ROUTE_SERVERS
 from cbw_api_toolbox.cbw_auth import CBWAuth
 from cbw_api_toolbox.cbw_objects.cbw_remote_access import CBWRemoteAccess
 from cbw_api_toolbox.cbw_objects.cbw_server import CBWCve
 from cbw_api_toolbox.cbw_objects.cbw_group import CBWGroup
+from cbw_api_toolbox.cbw_objects.cbw_users import CBWUsers
```
Comment (fwininger) [inline cbw_api_toolbox/cbw_api.py]: ordre alphabétique

File: documentation.md [hunk 3]
```diff
@@ -287,3 +287,23 @@ Send a GET request to `/api/v2/groups` to get informations about all groups
 | name                      | String        | Node name                         | "master"                                          |
 | url                       | String        | Node Url                          | "https://cyberwatch.fr"                           |
 | updated_at                | String        | Last modification of the node     | "2019-04-08T02:00:00.000+02:00"                   |
+
+### Users
+
+| Attribute                 | Type          | Description                       | Example of possible value                         |
+|---------------------------|:-------------:|:---------------------------------:|---------------------------------------------------|
+| email                     | String        | Email of user                     | "support@cyberwatch.fr"                           |
```
Comment (fwininger) [inline documentation.md]: test@example.com

File: documentation.md [hunk 4]
```diff
@@ -287,3 +287,23 @@ Send a GET request to `/api/v2/groups` to get informations about all groups
...
+| firstname                 | String        | First name of user                | "Cyberwatch"                                      |
+| id                        | Int           | User id                           | 25                                                |
+| locale                    | String        | Locale of user                    | "fr"                                              |
+| login                     | String        | Login email of user               | "support@cyberwatch.fr"                           |
```
Comment (fwininger) [inline documentation.md]: test@example.com

File: documentation.md [hunk 5]
```diff
@@ -287,3 +287,23 @@ Send a GET request to `/api/v2/groups` to get informations about all groups
...
+| name                      | String        | Name of user                      | "Cyberwatch"                                      |
+| server_groups             | Object        | Server Groups of user             | [server_groups](#Server-groups)                   |
+
+### Server groups
+
+| Attribute                 | Type          | Description                       | Example of possible value                         |
+|---------------------------|:-------------:|:---------------------------------:|---------------------------------------------------|
+| id                        | Int           | Server group id                   | 25                                                |
+| name                      | String        | Name of user                      | "Cyberwatch"                                      |
```
Comment (fwininger) [inline documentation.md]: Name of group

File: spec/test_cbw_api.py [hunk 6]
```diff
@@ -3,6 +3,7 @@
 from cbw_api_toolbox.cbw_api import CBWApi
 from cbw_api_toolbox.cbw_objects.cbw_server import CBWCve
 from cbw_api_toolbox.cbw_objects.cbw_group import CBWGroup
+from cbw_api_toolbox.cbw_objects.cbw_users import CBWUsers
```
Comment (fwininger) [inline spec/test_cbw_api.py]: ordre alphabétique

File: cbw_api_toolbox/cbw_objects/cbw_users.py [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+    """Users Model"""
+
+    def __init__(self,
+                 id="",  # pylint: disable=redefined-builtin
+                 login="",
+                 name="",
+                 firstname="",
+                 email="",
+                 locale="",
+                 server_groups=None,
```
Comment (jinxka) [inline cbw_api_toolbox/cbw_objects/cbw_users.py]: server_groups=[]

File: cbw_api_toolbox/cbw_objects/cbw_users.py [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+                 **kwargs): # pylint: disable=unused-argument
+        self.id = id  # pylint: disable=invalid-name
+        self.login = login
+        self.name = name
+        self.firstname = firstname
+        self.email = email
+        self.locale = locale
+        self.server_groups = CBWParser().parse(CBWUserServerGroups, server_groups[0]) if server_groups else None
```
Comment (jinxka) [inline cbw_api_toolbox/cbw_objects/cbw_users.py]: else []

**Pull request merged**